packages/create-vite/template-vanilla-ts/src/main.ts

```diff
@@ -8,7 +8,7 @@ document.querySelector<HTMLDivElement>('#app')!.innerHTML = `
       <img src="/vite.svg" class="logo" alt="Vite logo" />
     </a>
     <a href="https://www.typescriptlang.org/" target="_blank">
-      <img src="${typescriptLogo}" class="logo vanilla" alt="TypeScript logo" />
+      <img src="${typescriptLogo}" class="logo" alt="TypeScript logo" />
```

Comment:
I don't think this is the right change because removing `vanilla` here would give the TypeScript logo the same purplish glow as Vite's logo. The style for `.vanilla` should be fixed IMO.

The actual color shown appears to be for the yellowish hue for the vanilla JavaScript logo, as seen in the scaffolded style:

```css
.logo.vanilla:hover {
  filter: drop-shadow(0 0 2em #f7df1eaa);
}
```

And that's strange because the color is the correct bluish hue in:

https://github.com/vitejs/vite/blob/dd40a97a636139850f730bb11dbdb8e328389b7d/packages/create-vite/template-vanilla-ts/src/style.css#L55-L57

That seems to indicate that `create-vue` is copying over the vanilla JavaScript's `style.css` file instead of the TypeScript one.